IdokladSdk/Models/Account/AgendaContactGetModel.cs

```diff
@@ -25,6 +25,11 @@ public class AgendaContactGetModel
         /// </summary>
         public string Fax { get; set; }
 
+        /// <summary xml:lang="en">
+        /// Gets or sets a value indicating whether the company ID flag indication
```

Comment:
tento komentár nedáva v angličtine zmysel, zmenil by som to na "Gets or sets a value indicating whether the company has an ID"